----------------------------------------------------------------
Wed Jan 01 21:58:42 EST 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-016f-6430-dac1-00001d17ff90 
on database directory C:\Users\rajas\Documents\app\mydb with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@78308db1 
Loaded from file:/C:/Users/rajas/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=JetBrains s.r.o
java.runtime.version=11.0.5+10-b520.17
user.dir=C:\Users\rajas\Documents\app
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Wed Jan 01 22:10:13 EST 2020:
Shutting down instance a816c00e-016f-6430-dac1-00001d17ff90 on database directory C:\Users\rajas\Documents\app\mydb with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@78308db1 
Wed Jan 01 22:10:14 EST 2020 Thread[Thread-20,5,main] Cleanup action starting
ERROR XBM0I: Directory mydb cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:264)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Cleanup action completed
Wed Jan 01 22:10:14 EST 2020 Thread[Thread-20,5,main] Cleanup action starting
ERROR XBM0I: Directory mydb cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run$$$capture(FutureTask.java:264)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Cleanup action completed
